apps/recursive-routes/src/test-setup.ts

```diff
@@ -1 +1,14 @@
-import 'jest-preset-angular';
+import 'jest-preset-angular/setup-jest';
+
+import { getTestBed } from '@angular/core/testing';
```

Comment:
Why is this resetting of TestBed needed?

Reply:
This was added by the migrations. Most likely to add the `destroyAfterEach` flag